src/Configuration.php

```diff
@@ -126,6 +135,22 @@ public function setResultCacheImpl(Cache $cacheImpl): void
      */
     public function setSchemaAssetsFilter(?callable $callable = null): void
     {
+        if (func_num_args() < 1) {
+            Deprecation::trigger(
+                'doctrine/dbal',
+                'https://github.com/doctrine/dbal/pull/4620',
```

Comment:
:thinking:

Reply:
It was actually supposed to be the PR that deprecated `SQLLogger` but I'll update it to this one.